cwod_site/bioguide/management/commands/import_roles.py

```diff
@@ -57,5 +61,6 @@ def handle(self, *args, **options):
                 except IntegrityError:
                     continue
 
-            time.sleep(.55)
+            time.sleep(0.1)
```

Comment:
Did the propublica folks say if there's any rate limit to that api? Probably doesn't matter for this, but would be good to know.